### ISSUE TITLE
[EN] [PvP] [Mage] Mirror Image's [Gamebreaking]

### ISSUE THREAD

**Issue by Heroclassx** [Atlantiss/BugTracker#10192]
[//]: # (***************************************************************)
[//]: # (** DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED **)
[//]: # (***************************************************************)

[//]: # (On the Bugtracker all titles of reports must be written in English so that everyone, no matter which language they use everyday, could search and check if the problem already has been reported by another player. When describing the problem you can use English as well as Polish and in nearest future - others. It is important to determine at the beginning of a title in which language it is written by using symbols [EN] or [PL].)

[//]: # (Enclose links to things related to the bug using http://wowhead.com or http://cata.openwow.com)
[//]: # (Write your tickets according to the format:)
[//]: # ([EN][Quest][Zone] A Vision of the Past)
[//]: # ([EN][NPC] Thoralius the Wise)
[//]: # ([EN][Spell][Class] Frostfire Bolt)
[//]: # ([PL][Talent][Class] Brain Freeze)
[//]: # ([EN][Glyph][Class] Glyph of Frostfire Bolt)
[//]: # ([EN][Npc][Drop] Thoralius the Wise)
[//]: # ([EN][Web] Armory doesnt work)

**Description**: Mirror Images should stop casting on CC't targets (note: it wasn't like this pre Mirror image fix)

**Current behaviour**: Mirror Images keep casting into Polymorph, Dragonsbreath, etc..

**Expected behaviour**: Mirror Images should stop casting into CC

**Steps to reproduce the problem**:

1. Pop mirror images
2. use Poly morph on target getting attacked by mirrors
3. watch it break

**Revision**: Current server version (you can see current server revision by typing ".server info" command in the game chat)

**Comment by Illusie** [comment]
Are the IM frostbolts are mid-air otherwise it should broke.
If nah then no.

**Comment by Heroclassx** [comment]
Exactly, if the cast is already "done" and the mage casts poly while the frostbolts are mid air, they will ofcourse berak the CC. But since the rework, they **start** casting into CC't targets aswell.
